platform/persistence/platform-persistence-commands/src/main/java/org/codice/ddf/persistence/commands/AbstractStoreCommand.java

```diff
@@ -33,7 +33,7 @@
 
     protected PrintStream console = System.out;
 
-    protected Logger logger = LoggerFactory.getLogger(this.getClass());
+    protected Logger LOGGER = LoggerFactory.getLogger(this.getClass());
```

Comment:
private static final, might need to be protected static final if a child class uses the logger